mfa/errors.md

```diff
@@ -0,0 +1,523 @@
+{% multimethod %}
```

Comment:
Where did this file come from? It doesn't look like anything MFA related

Reply:
Snagged it from the other sections.  Not sure if we should keep it as a placeholder or not.